include/iceflow/consumer-tlv.hpp

```diff
@@ -169,22 +189,30 @@ class ConsumerTlv {
     m_scheduler.schedule(duration, [this] { sendAnchor(); });
   }
 
+  /**
+   * Receives the data from upstreams
+   * Pushes the data to the Input Queue of the Consumer
+   * Increase the data fetching/Interest sending rate - AIMD Congestion Control
+   * @param interest named data interest
+   * @param data actual data objects
+   */
```

Comment:
Formatting and text could probably be improved a little:

```suggestion
  /**
   * Handles data received from upstream producers.
   *
   * Pushes the data to the Input Queue of the Consumer, 
   * then gradually increases the data fetching/Interest sending rate
   * using AIMD Congestion Control.
   *
   * @param interest Named data interest
   * @param data Actual data objects
   */
```

What is meant by "Actual data objects", though? For AIMD you could also add a reference, I suppose, or explain the acronym.